apache/superset
# chore(python-testing): move memoized tests to unit tests

### SUMMARY
Today we have a huge suite of test that run for every type of database that superset support as meta database(MySQL postgres hive and etc')

This PR moves non DB related and basically pure unit tests to a lighter  github workflow
As a result of this 3 tests moved 2 tests won't be running 12 times (approx. number of database with python metrix)

A previous PR: https://github.com/apache/superset/pull/15473
I
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
* Python testing splited to  Python-Unit and Python-Integration
check that both pass
*  check locally that you are able to run the memoized_tests.py

### ADDITIONAL INFORMATION
